framework/OutStreams/FilePrint.py

```diff
@@ -16,20 +16,14 @@
 
 @author: alfoa
 """
-#for future compatibility with Python 3-----------------------------------------
-from __future__ import division, print_function, unicode_literals, absolute_import
-#End compatibility block for Python 3-------------------------------------------
-#External Modules---------------------------------------------------------------
 import os
-#External Modules End-----------------------------------------------------------
 
-#Internal Modules---------------------------------------------------------------
+from utils import InputData, InputTypes
 import DataObjects
-from .OutStreamManager import OutStreamManager
+from .OutStreamBase import OutStreamBase
 from ClassProperty import ClassProperty
-#Internal Modules End-----------------------------------------------------------
 
-class OutStreamPrint(OutStreamManager):
+class FilePrint(OutStreamBase):
```

Comment:
It's a base class instead of a manager now; it made sense to me.

It also made sure I found all the mentions of OutStreamManager everywhere in RAVEN.